app/src/main/res/drawable/ic_feedback.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Not sure but did you use the icon from the Figma design or the material design library? If it was from the Figma design, I think we can import the icon from the material design library directly in case Sarah wants the exact design from her design.

Reply:
The icon Sarah used was in the material design library, so i imported from there.